crates/frontend/src/components/info_modal.rs

```diff
@@ -0,0 +1,70 @@
+use leptos::*;
+
+#[component]
+fn info_section(
+    title: &'static str,
+    icon: &'static str,
+    content: String,
+) -> impl IntoView {
+    view! {
+        <div class="mb-5">
+            <h4 class="font-semibold mb-2 purple flex items-center">
+                <i class=format!("ri-{icon} mr-2")></i>
```

Comment:
tailwind does not work like this, take `ri-` also in the icon name
```suggestion
                <i class=icon />
```